### PR TITLE
CS:GO Admin

### PR DESCRIPTION
**Pull request by KillerWhalSam**

### CODE REVIEW

File: CS-GO-Administration-Guide/Getting Started.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+#Getting Started
 In this guide you will be shown how to use the In Game Admin menu. For CS:GO we have niether a Web Based Administration
 panel nor in game commands however we have an In Game Menu, this makes server administration easy. Although the menu is
 relatively simple we have created some guides which should make it easier for you to get started.
 
-##Want to get admin powers?
+###Want to get admin powers?
 If you are already a BF admin however want to be an admin on our CS:GO servers speak to @Kameeleon on Slack, have your
 Steam ID ready for us, you can find it [here.](http://steamidfinder.com/) If you are not an admin with any other server 
 Hosted by RocketBlast yet, we reccomend you apply [here.](http://bit.ly/adminrb)
```
Comment (edvinerikson) [inline CS-GO-Administration-Guide/Getting Started.md]: Change `RocketBlast` to `Rocket Blast`

**Comment by edvinerikson** [comment]
Rename `CS-GO-Administration-Guide` folder to `counterstrike` and move all battlefield files into a folder called `battlefield`.
Don't use uppercase letters and spaces in file names. 
Rename `In-Game-Admin-Menu.md` to `admin-menu.md`

Move/copy the existing `README.md` file into the `battlefield` folder. 
Create a new `README.md` file in the `counterstrike` folder, it should follow the style in the other `README.md` file (Table of contents section mostly)

In the end there will be three `README.md` files, one in the root directory (which holds a Table of content that points to the other files) and one in each sub directory.

**Comment by edvinerikson** [comment]
Change the `Table of contents` header to header 2 instead of 3 (remove one `#`). 
Convert the content of the `Table of contents` to a list. 
(put a `*` in front of each link e.g `* **[Violation guide](violations-guide.md)**`)

In the main `README.md`, links to other sections should deep link to the `README.md` file. e.g: `**[Counter Strike Admin Guides](counterstrike/README.md)**`

Change the header in the `README.md` file in the sub sections (battlefield and counterstrike) to reflect the current game. e.g: change Administration to Battlefield in the battlefield section and the same for the counterstrike (use `Counter-Strike` instead of `Battlefield`..) section.

counterstrike should be spelled `Counter-Strike` (and battlefield `Battlefield`, not `BF` or any other shortname) in text. (only use counterstrike in filenames and folders, etc)

**Comment by edvinerikson** [comment]
Looks good :+1:

**Comment by KillerWhalSam** [comment]
Thanks

**Pull request merged**